### DISCUSSION TITLE
Extension Figma Design

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#42] [dhaqsoPay Extension Developers]
Figma Design waakan:
https://www.figma.com/design/NYHcgHxMiTQpBw1w2nsN2h/dhaqsoPay?node-id=0-1&t=v9Oor9C9ElntzAxp-1

**Comment by hayle102** [comment]
Asxaabta waxaa soo dhamaaday design-ka Extension-ka waxaan rajayna bari in si toos ah dhisidiisa aan u bilowno

**linkiga design-ka waakan**
https://www.figma.com/design/NYHcgHxMiTQpBw1w2nsN2h/dhaqsoPay?node-id=0-1&t=lEIxKwmAg0ZKGULB-1

Sidoo kale waxa jira page ay kugu qoran yihiin guide-ka design-ka oo ay kamid yihiin Font-size, colors iyo cabirada waxaad ka heleysaa qaybta bidix ee figma.

<img width="236" alt="exten" src="https://github.com/user-attachments/assets/20b825ef-fed9-4dce-9a0d-209fa7d54159">


Shaqo wacan bari subaxna waxaan idin la soo wadaagi doona qaabka aan u qaybsan lahayn shaqada Insha allah.

@iftiiin 
@ayale3 
@faaraxcabdulaahi

**Reply by sharafdin** [reply]
bro project board habee kuna assign garee qof walba howshiisa lana wadaag sido kalena hag Insha'Allah!

**Comment by sharafdin** [comment]
dhalinyaro halkaan ku xeriira Insha'Allah! shaqo wacan

**Comment by hayle102** [comment]
Dhalinyaro waxaan sameeyay board aan ku qaybiyay shaqada qof walba waxa laga rabo iyadoo faahfaahsan issue ayuu ugu diyaarsanyahay waxaan rajaynaa inuu qof walba shaqadiisa qabsado sidoo kale xasuusnoow marka shaqada sidaas u bilowdo qaybtaada status-ka waa inad ka badashaa oo aad gaysaa **In progress** haddi aad dhamaysana **done** u badal status.

dadka shaqadooda ay ka mid ah popup kaliya diyaari popup-ka iyo overlay-giisa u dhaxeeya page-ka iyo popup-ka.

Good luck

**Reply by iftiiin** [reply]
Mahadsanid mohamed
Waa bilaabeenaa i.a

**Reply by faaraxcabdulaahi** [reply]
> Dhalinyaro waxaan sameeyay board aan ku qaybiyay shaqada qof walba waxa laga rabo iyadoo faahfaahsan issue ayuu ugu diyaarsanyahay waxaan rajaynaa inuu qof walba shaqadiisa qabsado sidoo kale xasuusnoow marka shaqada sidaas u bilowdo qaybtaada status-ka waa inad ka badashaa oo aad gaysaa **In progress** haddi aad dhamaysana **done** u badal status.
> 
> dadka shaqadooda ay ka mid ah popup kaliya diyaari popup-ka iyo overlay-giisa u dhaxeeya page-ka iyo popup-ka.
> 
> Good luck

Eng aad baad umahadsantahay

**Comment by ducesane** [comment]
Howsha waa bilaaabatay

**Comment by ducesane** [comment]
![Captureerew](https://github.com/user-attachments/assets/276e217c-71ef-4cfe-b4b4-82706c042e12)
Markaa sameynaa  page-kaan oo kale  pop-upka kaliyaa ma la sameynaaa

**Reply by iftiiin** [reply]
Haa

**Reply by ducesane** [reply]
waa war fiican @iftiiin

**Comment by hayle102** [comment]
Dhalinyaro waan arkaa howsha waad bilowdeen shaqa wacan sidoo kale discussion-kaan inaad kuwada xiriirtaan isku daya

**Reply by iftiiin** [reply]
Diyaar waaye.
Qofkasta pop-upkiisa markuu sameeyo kadib aa mideenaa soomaha?

**Reply by hayle102** [reply]
Haa Eng marka wada dhameeno ayaa ka mideyn doona insha allah

**Reply by faaraxcabdulaahi** [reply]
> Dhalinyaro waan arkaa howsha waad bilowdeen shaqa wacan sidoo kale discussion-kaan inaad kuwada xiriirtaan isku daya

Hagaag eng

**Comment by iftiiin** [comment]
Asxaabta xagee wax marineysaan?

**Reply by hayle102** [reply]
Eng project dhisidiisa way socotaa insha allah waxaan rajaynaa inaan isbuucaan ka dhmaano adiga halkeed wax marsiine maka dhamaatay qaybtaada

**Reply by iftiiin** [reply]
Ha waaka dhamaday qeyteda

**Reply by hayle102** [reply]
Oky Eng qayb kale ayaan kugu soo assign garaynaa insha allah

**Reply by iftiiin** [reply]
Ok

**Reply by hayle102** [reply]
Eng qayb cusub ayaan kugu soo assigne gareyay

**Comment by hayle102** [comment]
Dhalinyaro inta aanan ka ahayn @iftiiin  inta kale xagee wax marsiinaysaan

**Reply by faaraxcabdulaahi** [reply]
> Dhalinyaro inta aanan ka ahayn @iftiiin inta kale xagee wax marsiinaysaan

Eng 10% baa ii harsan qaybtayda

**Reply by ducesane** [reply]
Qeybteeda waa so sameyay.

**Reply by hayle102** [reply]
Aaad baad u mahadsantihiin

**Comment by iftiiin** [comment]
![image](https://github.com/user-attachments/assets/63d8add9-1b5a-45a8-ad31-9a0a98616506)
@hayle102 Kwrn hadii isku mid ka dhigno oo calamad qalad cml ku darno 
si ee hal model noogu noqdan

**Reply by hayle102** [reply]
Waa si fican sxb wan ku dari karnaa

**Comment by hayle102** [comment]
Asxaabta dhankayga waxaan dhameeyay labada page ee kala signup and signin factionalaties-kooda iyo wax walba oo quseeya iyaga.

![image](https://github.com/user-attachments/assets/a2f2170c-f265-434e-98e9-9ac3ca725a6f)
![image](https://github.com/user-attachments/assets/10be10ef-4d25-4778-bcfe-33c5c5335aa7)

cabirka extension-ka sidee ayuu idin noqday

**Reply by iftiiin** [reply]
Ok waye qebtakalana waakaa sugeenaa.
hada waxaa soo sameynaa pull request oo branch extension firiya i.a

**Reply by hayle102** [reply]
Insha allah Eng

**Comment by hayle102** [comment]
Asxaabta waxaan pull request ka soo sameeyay auth pages oo signin iyo sign up pages fiiriya kadib wixi comments wax ka badal ah isoo celiya haddi kalana hala aqbalo
insha allah

**Reply by iftiiin** [reply]
Diyaar waaye eng.

**Reply by iftiiin** [reply]
Get started-ka logada iyo mida qoralka leh  saaid iskuugu dhawyihiiin xogaa inaa space galiso soo jeedin lahaa
sidokale qofka markuu signup samenayo number intuu rabo uu galin karaa inaa meshaasna xadido oo 7 number kaliya aaka dhigto inta kale waa ok.

**Reply by hayle102** [reply]
Okey, Eng mida logada aniiga qaybta css si kam ah ku delete  gareeyay mida numberka ahna patterns ayaan ku dari doona insha allah thank you Eng

**Comment by hayle102** [comment]
Waxaan soo sameeyay pull request cusub hala iga fiiriyo

**Reply by iftiiin** [reply]
Ok

**Comment by iftiiin** [comment]
Waxaa soo sameyay pull request qeybta payment fiiriya i.a
@ayale3 and @faaraxcabdulaahi waxaa u malenaa qeybtiina inaa sameyseen pull request kusoo sameya branch entension-ka

**Reply by ducesane** [reply]
waa soo sameyay labo Pull request

**Reply by faaraxcabdulaahi** [reply]
> Waxaa soo sameyay pull request qeybta payment fiiriya i.a @ayale3 and @faaraxcabdulaahi waxaa u malenaa qeybtiina inaa sameyseen pull request kusoo sameya branch entension-ka

Eng waan soo sameeyay pull requestiga

**Reply by iftiiin** [reply]
Mainka aa request kusamesen extension branch aa marka hore iskuugu geynenaa engs

**Comment by hayle102** [comment]
Eng pull request branch extension-ka ka sameeya ma ahan midka main-ka marka hore aan extension iskugu geeyno

**Reply by ducesane** [reply]
![Screenshot_20241128_104338_GitHub.jpg](https://github.com/user-attachments/assets/22f2ea73-62aa-4fd2-900f-a260db6f3ea1)

![Screenshot_20241128_104435_GitHub.jpg](https://github.com/user-attachments/assets/4e300121-0867-4f09-91bb-51790c2c0875)



Branch ka extention ka hadaa pull request ku sooo sameeyo conflicts ayaa jiro hadaaa

**Reply by ducesane** [reply]
@hayle102   conflicts waa xaliyay iga aqbal Puill request-ka

**Reply by iftiiin** [reply]
@ducesane Waxaa aqbalay pull request-ka change pin kadib request-ka kale waxaa ka dhacay conflicts eng fiiri

**Reply by ducesane** [reply]
Hada eeg @iftiiin

**Reply by iftiiin** [reply]
Ok waaye waa aqbalay

**Comment by faaraxcabdulaahi** [comment]
@hayle102 Eng pull request baan soo dirsaday adigana waxaan kaa dhigay reviewer, marka aqbal

**Reply by hayle102** [reply]
Okey eng waa diyaar Insha allah

**Comment by hayle102** [comment]
@faaraxcabdulaahi  Eng review ayaan ku soo siiyay fiiri pull request soo codsatay

**Reply by faaraxcabdulaahi** [reply]
> @faaraxcabdulaahi Eng review ayaan ku soo siiyay fiiri pull request soo codsatay

**Eng ilaa hada weli pull requestigii waa taaganyahay**

![image](https://github.com/user-attachments/assets/bf668c8c-1037-42a9-b542-9dc2e228d750)

Qofka pull requestiga codsanaya inuu awood u yeesho inuu isagu merge oron karo waa laga block gareeyay sababtana branch rule baa lagu  xriray taa waa fahansanahay, **balse eng adigu ka reviewer ahaan, codeka markaad review tiraahdid kadibna approve,  _Awood ma  uleedahay in merge tiraahdo ?_**

**Reply by hayle102** [reply]
Eng aniga merge iraahdo awoodba uma lihi sidaas ayaan review kaaga soo bixyay

**Reply by iftiiin** [reply]
@faaraxcabdulaahi waxaa samesa pull request-gas inaa close iska dhahdo mid cusub sameso hada extension iyo main aa request ka samenosa.
Markaa sameneso request-ga base waxaa ka dhigtaa extension branch compare topUp branch markaas adi xitaa waa aqbali kartaa

**Comment by iftiiin** [comment]
@hayle102 waqtiga gabogabo waaye waxyaabha noo dhiman nagu kla assign garee

**Reply by hayle102** [reply]
Boss aniga dashboardka ayaan hadda ii dhamaan raba wan soo diraaya kadib extension isku wada gaynaynaa talaadada waa la isku xirxiraaya qolada backend waa diyaar koleey qaybtooda

**Reply by iftiiin** [reply]
Inaa anga hal file  isugu geyno managala rabo mia ?
Sidoo kale light mode inaa sameno soomaaha?

**Reply by hayle102** [reply]
Eng kaasi I think qorshaha waa lag saaray dark mode ayaa default ahaan loo qaadana soo tii la yiri.
Extension iskugu geyneynaa kadib teamka ayaa main gayn doona kadib ayaa hal file laga dhihi doona

**Reply by iftiiin** [reply]
Taas waxaa u fahmay dark inuu marka hore extension-ka kusoo kaco kadib user-ka uu doorto mode siduu noqonaayo may be wa iga saxanthy

**Comment by hayle102** [comment]
Asxaabta cabirka sugaysay waan idin la soo wadaagay ka fiiriya discussion-ka #64

**Comment by hayle102** [comment]
Asxaabta waxaan soo sameeyay pull request wuxu leeyahay conflict ayaa jiro mid ayaan ku xaliyay github dhexdiisa oo inta sababaysa conflict ayaan delete gareeyay mid kale ayuu wali ku haystaa oo ah dhanka file-ka style oo style cusub oo aan hadda ku soo push gareeyay branch-gayga kadib aan request u codsaday extension ayuu leeyahay wuxuu sababaya conflict. hala fiiriyo wixii review ah hala ii dhaafo.

**Reply by iftiiin** [reply]
request-ga waa aqbalay lkn style.css file-kaas waxaa iska badlay waxaana keenay conflicts badanaa ka dhacay @hayle102 iyo @ducesane 
labadina isla isticmaaleysen labo file inaa kla sameysataan amee roonthy.
hada branch extesion hadii la tago design-ka dhinacyadaas wuu isbadaly

**Reply by hayle102** [reply]
oo eng files style.css ayaa ducaysane ila isticmaalay oo igula magac ah miyaa hadda ma xalisay oo code css maku jiraa extension-ka

**Reply by hayle102** [reply]
> request-ga waa aqbalay lkn style.css file-kaas waxaa iska badlay waxaana keenay conflicts badanaa ka dhacay @hayle102 iyo @ducesane labadina isla isticmaaleysen labo file inaa kla sameysataan amee roonthy. hada branch extesion hadii la tago design-ka dhinacyadaas wuu isbadaly

Eng waan soo arkay file style.css waxaa isu tagay style aniga isticmaalay iyo kii duceysane hadda duceysane ayaa waxa uu u baahana inuu file-kaas ku soo xirmo sidiisa kale code-ka waa kuwada jiraa

**Reply by iftiiin** [reply]
Ok waaye markaas

**Reply by ducesane** [reply]
@iftiiin @hayle102   hada waa sooo xaliyay ciladaha ka imaaday , Pull Request-ga aan ku soo sameyay Extention-ka  ee aqbal , file-ka dash.js ne wax yar oo js aya ku soo qoray marka  change pin icon click lagu soo siiyo  ina oo aado Change Pin screen Page

**Reply by hayle102** [reply]
Okey eng modal-ka ma raacday width aan calaamadeeyay

**Reply by ducesane** [reply]
Haaa cabirka aa shegtay aa qaatay

**Comment by hayle102** [comment]
Asxaabta hadda waxaan filaaya qaybta front-end ee extension inaan ka dhamaanay yaa ay wax ku harsanyihiin

**Reply by iftiiin** [reply]
Modals-ka inaa same width ka dhigno kaliyaa dhiman

**Reply by hayle102** [reply]
Midaasi waan soo sheegay width saxda ah disscusion ka eega

**Reply by iftiiin** [reply]
Ok waa arkay waa soo saxayaa width-ka

**Comment by hayle102** [comment]
Asxaabta branch-gayga oo ahaa auth-dash waxaa la iskugu keenay dhamaan qaybaha project-ga sidee ku dhacday hadda ayaan waxaan is iri logada light u badal markaan maagay inaan soo push gareeyo ayuu igu dhagay waxayna ahayd inan shalay conflict mid ka mid ah aan ku xaliyay github tool kadib aayaan pull soo sameyay waxaaba isoo galay dhamaan inta ka shaqaynayseen sidee ku dhacday.

**Reply by iftiiin** [reply]
Shalay reques-gaa soo sameysay conflicts uu lahaa anaa isla github ku xaliyay! 
in arintaas ee sababtay ma sheegi karo.
Hada wali masoo push gareyn kartid mia?

**Reply by hayle102** [reply]
maya eng hadda waxba ma soo push garayn rabo lkin wan ku wareeray

**Comment by hayle01** [comment]
@iftiiin eng  ayaan dhoweed marka samaysay conflict merge branch-gayga waxaa yimid all pages ka shaqaynayseen wuxu ii sababay qaybta style inuu wax iga qalday hadda wan soo saxay waxan soo dalbaday pull request wuxu sheegaya conflict waa isla file-kii style-ka

**Reply by iftiiin** [reply]
Eng file-kaas adi iyo ayaanle aa isla isticmaalooyo intaa push soo sameyn git pull ma dhahday

**Reply by hayle01** [reply]
Maya hadda is iri pull ma dhihin  hadda xali bal eena aqbal mahadsanid

**Reply by iftiiin** [reply]
Dhankaaga eeka fududahy madaama hada wax kusoo dartay

**Reply by hayle01** [reply]
Okey

**Comment by hayle01** [comment]
Asxaabta waxaan pull request ka soo sameeyay waxaan iskugu geeyay dhamaan models-kii oo waxaan ka dhigay mid ka soo wada kacaya hal page oo ah page-ka dashboard iyadoo la isticmaalayo hal modal kaliya waxaa la badali doona contents-ka, inputs-ka iyo validations-ka iyadoo loo eegayo xilligaas action-ka dhacaayo e.g topup, check balance, iyo change PIN

marka review hala soo siiyo ka dib hala aqbalo

**Reply by iftiiin** [reply]
Ok eng

**Reply by iftiiin** [reply]
Waxaa ku weydina qeybta payment request modal gooni ah  aa u deysay mia?

**Reply by hayle01** [reply]
maya eng qaybtaas isla kan ayaa la isticmaali doona lkn request ayuu u baahanyahay marka la diyaariyo ayaa lagu xiri doona insha allah

**Comment by hayle01** [comment]
Asxaabta pull request cusub sameeyay hala review gareeyo
